### ISSUE TITLE
about learning rate

### ISSUE THREAD

**Issue by martinzhubin** [policeme/transformer-pointer-generator#29]
The "lr" param in params.py is not used. 

In model.py:
lr = noam_scheme(self.hp.d_model, global_step_, self.hp.warmup_steps)
optimizer = tf.train.AdamOptimizer(lr)

And the "noam_scheme" function is different from the official Transformer code. Can you explain this?

**Comment by xiongma** [comment]
this is just use warmup strategy in lr
